stackit/internal/services/cdn/distribution/resource.go

```diff
@@ -256,6 +262,13 @@ func (r *distributionResource) Schema(_ context.Context, _ resource.SchemaReques
 								Description: schemaDescriptions["config_backend_origin_request_headers"],
 								ElementType: types.StringType,
 							},
+							"geofencing": schema.MapAttribute{
+								Description: "A map of URLs to a list of countries where content is allowed.",
+								Optional:    true,
+								ElementType: types.ListType{
+									ElemType: types.StringType,
+								},
```

Comment:
add here a validator, otherwise it's possible to configure an empty map for geofencing which results in an error
```suggestion
								Description: "A map of URLs to a list of countries where content is allowed.",
								Optional:    true,
								ElementType: types.ListType{
									ElemType: types.StringType,
								},
								Validators: []validator.Map{
									mapvalidator.SizeAtLeast(1),
								},
```